Week1/homework/world-queries.txt

```diff
@@ -37,4 +39,199 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
 | Heerlen           |
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
+
+2. What is the population of Rotterdam ?
+
+SELECT Name, Population FROM city 
+WHERE city.Name = 'Rotterdam';
++-----------+------------+
+| Name      | Population |
++-----------+------------+
+| Rotterdam |     593321 |
++-----------+------------+
+1 row in set (0.01 sec)
+
+3. What's the name of all the countries on the continent ‘North America’ ? 
+
+SELECT Name,Continent FROM country 
+WHERE country.Continent = 'North America';
```

Comment:
👍

Keep in mind that the question asked you for the name of the countries. It didn't ask you to return the name and continent - just the name.
You got the query exactly correct, but `SELECT Name FROM ...` would be closer to what the question actually asked.

Reply:
Hi @arasmr and @hackoor83, did you commit the fixes or comment on each other?